src/dag/dag.cpp

```diff
@@ -293,7 +295,8 @@ DagManager::DagManager(blk_hash_t const &genesis, addr_t node_addr, std::shared_
       db_(db),
       anchor_(genesis),
       period_(0),
-      genesis_(genesis) {
+      genesis_(genesis),
+      log_time_(log_time) {
```

Comment:
what is this log_time good for ?

Reply:
this log_time is some performance logger used across different classes. I am not sure if we actually use it for something. We need to check with everyone and remove all instances if it is not used